docs/proposals/algorithms/lifelong-learning/Real-Time IoT Perception Systems Based on Edge-Cloud Collaboration with Large Foundation Models.md

```diff
@@ -0,0 +1,121 @@
+
+<img src="images/云边协同改进.png" width="50%" >
+
+The benefits of cloud-edge collaborative reasoning are not only reflected in the optimization of computing resources but also in improving the real-time performance and stability of the system. The large model in the cloud can continuously improve its recognition ability through learning and training, while edge devices can process simple samples in real-time, thereby improving the overall system's response speed. At the same time, when the small model on the edge device encounters issues or requires updates, remote model updates can be performed through communication with the cloud to maintain system stability.
+
+## 3 Design Detail
+
+### 3.1 Overview architecture
+
+The overall architecture of the Cloud-edge Collaborative IoT Real-time Perception System is shown in the figure below. This system architecture considers applications in typical lifelong learning scenarios. Edge devices continuously collect images of the environment they are located and efficiently process these images using the Cloud-edge Collaborative Real-time Perception System. At the same time, the edge model is continuously trained through lifelong learning, based on the exploration of hard sample patterns, to ensure that the model can continuously adapt to the dynamic changes in the edge environment.
+
+![joint-inference](images/joint-inference.jpg)
+
+The overall workflow of the system is as follows:
+
+1. Edge nodes collect samples for inference.
+2. Edge nodes use deployed Edge Model to infer these samples and obtain edge inference results.
+3. The edge inference results are sent to the Hard Example Mining module.
+4. The Hard Example Mining module classifies the samples based on the inference results, labeling them as Hard samples and Easy samples.
+5. Easy samples are directly output with their edge inference results, while Hard samples are uploaded to the cloud inference module on the cloud node.
+6. In the cloud inference module, hard samples are inferred by the SAM model to obtain the cloud inference mask, and the edge inference results are obtained by the Edge Model.
+7. In the cloud inference module, the cloud inference results are obtained by combining the cloud inference mask and the edge inference results.
+8. Edge nodes download the cloud inference results, while cloud node storing hard samples and their cloud inference results in the cloud node's replay buffer.
+9. Edge nodes output the cloud inference results, while the cloud node performs lifelong learning training on the edge model based on the samples in the cloud node's replay buffer.
+10. The cloud node synchronizes the updated Edge Model to the edge nodes.
+
+![plugin-workflow](images/plugin-workflow.jpg)
```

Comment:
As mentioned above, the training modules are also absent in the workflow.

Reply:
> As mentioned above, the training modules are also absent in the workflow.

I have added training modules in the workflow.